### PR TITLE
BOAC-2646 Display modifications for inactive and completed academic careers

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/BOAC-2646

### CODE REVIEW

File: src/components/course/CourseStudents.vue [hunk 1]
```diff
@@ -35,19 +35,37 @@
...
       </div>
-      <div>
-        <div v-for="major in row.item.majors" :key="major" class="student-text">{{ major }}</div>
+      <div
+        v-if="displayAsCoeInactive(row.item)"
+        :id="`student-${row.item.uid}-coe-inactive-for-${section.termId}-${section.sectionId}`"
+        class="student-sid red-flag-status">
+        CoE INACTIVE
+      </div>
+      <div v-if="row.item.academicCareerStatus != 'Completed'">
```
Comment (johncrossman) [inline src/components/course/CourseStudents.vue]: @paulkerschen - should `!=` be `!==`? I see three of 'em.

File: src/components/course/CourseStudents.vue [hunk 2]
```diff
@@ -35,19 +35,37 @@
...
+        <div>
+          <span class="student-text">{{ row.item.level }}</span>
+        </div>
+        <div>
+          <div v-for="major in row.item.majors" :key="major" class="student-text">{{ major }}</div>
+        </div>
+      </div>
+      <div v-if="row.item.academicCareerStatus == 'Completed'">
```
Comment (johncrossman) [inline src/components/course/CourseStudents.vue]: @paulkerschen same question on ` == `

File: src/components/course/CourseStudents.vue (hunk 1 above)
Comment (paulkerschen) [reply src/components/course/CourseStudents.vue]: @johncrossman Thanks - I'm surprised the linter didn't catch those.

**Comment by johncrossman** [comment]
Thanks @paulkerschen  Waiting on T

**Pull request merged**